tests/neural_compressor/test_optimization.py

```diff
@@ -278,6 +307,41 @@ def test_dynamic_diffusion_model(self):
         # Compare model outputs
         self.assertTrue(np.allclose(loaded_pipe_outputs, outputs, atol=1e-4))
 
+    def test_quantize_text_generate_model_quantize(self):
+        model_id = "EleutherAI/gpt-neo-125m"
+        set_seed(42)
+        model = AutoModelForCausalLM.from_pretrained(model_id)
+        tokenizer = AutoTokenizer.from_pretrained(model_id)
+        tokens = tokenizer("This is a sample", return_tensors="pt")
+        input_ids = model.dummy_inputs["input_ids"]
+        input_bs, input_len = input_ids.shape
+        past_key_values = generate_dummy_past_key_values(input_bs, model)
+        attention_mask = torch.ones(input_bs, input_len + 1)
+        attention_mask[:, 0] = 0
+        example_inputs = (
```

Comment:
is this specified so that pkv can be leveraged ?

Reply:
removed it, since the Ipex quantization is not ready now.